src/App.js

```diff
@@ -0,0 +1,63 @@
+// import logo from "./logo.svg";
+import logo from "./images/ironhack-logo-xs.png";
+import menu from "./images/menu-top-xs.png";
+import "./App.css";
+import card1 from "./images/icon1.png";
+import card2 from "./images/icon2.png";
+import card3 from "./images/icon3.png";
+import card4 from "./images/icon4.png";
+
+function App() {
+  return (
+    <>
+      <div className="App">
+        <img src={logo} alt="example" className="navbar-col navbar-item" />
```

Comment:
I would use a meaning full alt to increase usability of your app

```suggestion
        <img src={logo} alt="navbar icon" className="navbar-col navbar-item" />
```